static/js/zamboni/devhub.js

```diff
@@ -691,6 +691,91 @@ function initVersions() {
             $('.current-version-warning', this).toggle(is_current);
             return true;
         }});
+
+    function loadReviewHistory(div, nextLoad) {
+        div.removeClass("hidden");
+        div.prop("style", "");
```

Comment:
It's some helpful jquery(?) 'feature' that set `"display:none"` to every element with a class of `hidden`.  If you know how to turn it off that would be ideal because it's super annoying.